tests/kconfig/app.config.test

```diff
@@ -0,0 +1,2 @@
+CONFIG_MODULE_EXTERNAL_MODULE_1=y
```

Comment:
The solution seems to work fine, but the hash check fails, probably because when `TEST_KCONFIG=1` `app.config` is not loaded (`app.config.test` is instead). So I think we would nee to add `app.config` to `KCONFIG_ADD_CONFIG`

Reply:
You mean:

```
diff --git a/tests/kconfig/Makefile b/tests/kconfig/Makefile
index 5e73126239..10cf5a2abd 100644
--- a/tests/kconfig/Makefile
+++ b/tests/kconfig/Makefile
@@ -4,4 +4,8 @@ USEMODULE += external_module_1
 USEMODULE += external_module_2
 EXTERNAL_MODULE_DIRS += external_modules
 
+ifeq (1, $(TEST_KCONFIG))
+  KCONFIG_ADD_CONFIG += $(APPDIR)/app.config.test
+endif
+
 include $(RIOTBASE)/Makefile.include
```

Reply:
Hmm I think when `TEST_KCONFIG` you will need to add `app.config`, because `app.config.test` is added automatically already.

Reply:
Yes sorry app.config